### PR TITLE
Just autogpt move

### PR DESCRIPTION
**Pull request by ryanpeach**

### CODE REVIEW

File: autogpt/__main__.py
Comment (nponeccop) [inline autogpt/__main__.py:275]: You can still get it slimmer by removing this type of changes

File: autogpt/__main__.py
Comment (nponeccop) [inline autogpt/__main__.py:143]: Well, you can still make it slimmer by removing changes like this one

**Review by nponeccop — COMMENTED** [review]
Good idea. It isn't perfect but much more focused.

File: autogpt/__main__.py
Comment (ryanpeach) [reply autogpt/__main__.py:275]: I'm not going through the pr and removing all the whitespace changes. That would be very manual.

Comment (ryanpeach) [reply autogpt/__main__.py:275]: Maybe if the whitespace PR was merged first this diff would get smaller.

File: autogpt/chat.py
Comment (dhensen) [inline autogpt/chat.py:143]: This looks my code edit, but I get zero attribution with this new PR you've proposed.
Who cares anyway.. the main importance is that it get's merged, but it's a shame that you dont give attribution to people's work @ryanpeach or maybe messed up some git commands, who knows. Anyways, I don't think creating yet another PR for the same thing makes sense if your goal is getting it merged. Or is this PR doing something new? Can't get it out of the supreme PR description ;)

Comment (ryanpeach) [reply autogpt/chat.py:143]: My point wasn't to cut you out. How would I add attribution other than the fact I have mentioned it several times in the original PR.

Comment (Torantulino) [reply autogpt/chat.py:143]: @ryanpeach

If PRs are not atomic, focused on a single thing, they put me in a difficult position when it comes to considering merging them.

I've been urged many times by members of the community that merging PRs that contain multiple changes or combined PRs sets a bad precedent and crate a broken git history. Additionally they make it difficult to remove problem causing PRs at a later date if necessary.

Atomic PRs that don't sneak in any extra little changes, that are well documented and only have diffs on the intended lines tend to get merged as soon as they've been tested - so long as they align with the project's direction.

Comment (Torantulino) [reply autogpt/chat.py:143]: Ultimately this is my fault for not making the guidelines for contributors clear. 
I'll work on this.

File: autogpt/__init__.py
Comment (Torantulino) [inline autogpt/__init__.py:1]: Is this intentionally empty?

File: autogpt/commands.py
Comment (Torantulino) [inline autogpt/commands.py:125]: Why does this PR try to sneak in a change away from using Duck Duck Go?

The PR that changed that allowed us to start getting much better results and less errors.

File: autogpt/commands.py
Comment (Torantulino) [inline autogpt/commands.py:166]: Again, this is removing another really powerful feature, and it's not even mentioned in the PR description or title.

File: autogpt/commands.py
Comment (Torantulino) [inline autogpt/commands.py:179]: Another feature this PR removes, unrelated to the topic of change.

**Review by Torantulino — CHANGES_REQUESTED** [review]

File: autogpt/commands.py
Comment (nponeccop) [reply autogpt/commands.py:179]: This is exactly why we want PRs to be focused. It's easy to overlook.

**Comment by nponeccop** [comment]
@ryanpeach Now it's also outdated. I guess the unrelated changes weren't intentional but creeped in with many merges into the original commit. That's why I always rebase and squash my commits. It's easier to see the diff you introduce.

File: autogpt/__init__.py
Comment (ryanpeach) [reply autogpt/__init__.py:1]: Yes, that is usually how init files are. It makes the folder a module.

**Comment by ryanpeach** [comment]
I’m going to close these PRs and stop contributing until this very minor and very necessary feature is complete. There are literally 3 PRs doing the exact same thing. Pick one, add an init file to the scripts folder, do it yourself, do something. But I’m out until it’s resolved.

**Comment by nponeccop** [comment]
Note that we don't merge big features without CI. Most of the today's 13 PRs merged were documentation or very trivial things, with the debug flag and cfg being the two cases of minor and quickly resolved breakage.

We can always ask @Torantulino to define our strategy. I don't mind doing whatever the maintainer wants.

**Pull request closed without merging**
